tests/clocks/input_attr_not_clock/README.rst

```diff
@@ -0,0 +1,27 @@
+Force input as regular input by setting the CLOCK attribute
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+`input wire a` should be detected as a clock because it drives the flip flop. However, it has the attribute CLOCK set to 0 which should force it to be a regular input.
+
+.. symbolator:: ../../../tests/clocks/input_attr_not_clock/block.sim.v
```

Comment:
Is there a reason these can't just be `block.sim.v`? Is it because you use an `.. include::` in the docs directory?

Reply:
Yes so sphinx looks for the verilog file based on where README.rst was included. Can't seem to find a way for it to search based on the original location of README.rst.

Reply:
But this does make the README.rst look bad when viewed in GitHub. I guess we should think of a different way to do this?

Reply:
@mithro I think the best way is to create a symlink to each of block.sim.v files inside the docs directory. What do you think?

Reply:
Ok, not possible to create a symlink to block.sim.v they all will have to be under the same directory. I also tried it with one of the files and somehow sphinx does not manage to read it. 

@mithro Any ideas? Or just leave it as how it is now?

Reply:
@daniellimws -- I'm going to have to look at the closer and I'm afraid I don't have the time for now.

Rather than blocking this, let's land with the ugly stuff and create a GitHub issue to figure out how to solve this problem?

Reply:
Ok yes we should